docs/tutorials/scrape_websites_using_the_sitemap.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Scrape websites using the sitemap
+description: The sitemap.xml file is a jackpot for every web scraper. Take advantage of this and learn a much easier way to extract data from websites using the Apify SDK.
+menuWeight: 3.7
+paths:
+    - tutorials/scrape-websites-using-the-sitemap
+---
+
+# Scrape websites using the sitemap
+
+Let's say we want to scrape a database of craft beers ([brewbound.com](https://www.brewbound.com)) before the summer season starts. If we are lucky, the website will contain a sitemap at [https://www.brewbound.com/sitemap.xml](https://www.brewbound.com/sitemap.xml).
+
+> Check out out [Sitemap Sniffer](https://apify.com/vaclavrut/sitemap-sniffer) tool, which can discover sitemaps in hidden locations.
+
+## [](#the-sitemap) The sitemap
+
+The sitemap is usually located at the path `/sitemap.xml`. It is always worth trying that URL, as it is rarely linked anywhere on the site. It usually contains a list of all pages in [XML format](https://www.w3.org/standards/xml/core).
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>http://www.brewbound.com/advertise</loc>
+        <lastmod>2015-03-19</lastmod>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+    ...
+```
+
+The URLs of breweries are in the form
+
+```cURL
+http://www.brewbound.com/breweries/[BREWERY_NAME]
+```
+
+and the URLs of craft beers are in the form
+
+```cURL
+http://www.brewbound.com/breweries/[BREWERY_NAME]/[BEER_NAME]
+```
+
+They can be matched with the following regular expression.
+
+```cURL
+http:\/\/www\.brewbound\.com\/breweries\/[^\/]+\/[^\/<]+
```

Comment:
It looks they started using SSL on their web. Maybe just for sure fix the regexp to handle https as well. If they migrated sitemap to https the guide will still work.
`http(s)?:\/\/www\.brewbound\.com\/breweries\/[^\/]+\/[^\/<]+`